discussions/api/v1/views.py

```diff
@@ -15,3 +21,16 @@ class UserDetailView(generics.RetrieveAPIView):
     # shouldn't be necessary in theory but DRF is returning Not Found otherwise
     def get_object(self):
         return User.objects.get(external_id=self.kwargs['external_id'])
+
+
+class SubscriptionsDetailView(generics.RetrieveAPIView):
+    """
+    API endpoint that allows users to be viewed.
+    """
+    serializer_class = SubscriptionsSerializer
+    queryset = Subscriptions.objects
+    lookup_field = '_id'
```

Comment:
since we seem to need `get_object()` the `queryset` and `lookup_field` don't need to be defined.